backend/core/db/models/congregation.go

```diff
@@ -39,7 +46,8 @@ type Congregation struct {
 	// Should not be modified/retrieved directly. Only through GetPhones/SetPhones
 	PhoneNumbers datatypes.JSON `json:"phoneNumbers"`
 
-	Users []User `json:"users" gorm:"foreignKey:CongregationID"`
+	Users            []User                 `json:"users" gorm:"foreignKey:CongregationID"`
+	InformationBoard []InformationBoardItem `json:"informationBoard" gorm:"foreignKey:CongregationID"`
```

Comment:
This foreign key needed (which was causing the error) needs to be linked to that Congregation identifier.